Automattic/jetpack
# Form Block: Allow Changing "Message Sent" Phrasing

Fixes #19640

#### Changes proposed in this Pull Request:

* When a user submits a form created with the Form Block, "Message Sent" will appear. The text below can already be customised, but this PR would allow users to change the phrasing in that heading.

#### Jetpack product discussion



N/A - this PR is in response to the original issue (and it's something I'd like to see too!)

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






Create a Form Block and verify that there's a new "Message Heading" option

<img width="266" alt="Screenshot 2021-05-01 at 14 38 02" src="https://user-images.githubusercontent.com/43215253/116784228-09693d00-aa8b-11eb-8a3e-537b6dbab28d.png">

Enter some text there, and submit the form as a user. You should see that text appear once that form has been successfully submitted.

<img width="656" alt="Screenshot 2021-05-01 at 14 39 25" src="https://user-images.githubusercontent.com/43215253/116784230-0d955a80-aa8b-11eb-9f36-7ddb07f06d1c.png">
